pyquil/api/_quantum_computer.py

```diff
@@ -669,6 +671,40 @@ def get_qc(name: str, *, as_qvm: bool = None, noisy: bool = None,
 def local_qvm() -> Iterator[Tuple[subprocess.Popen, subprocess.Popen]]:
     """A context manager for the Rigetti local QVM and QUIL compiler.
 
+    .. deprecated:: 2.11
+        Use py:func:`local_forest_runtime` instead.
+    """
+    warnings.warn(DeprecationWarning("Use of pyquil.api.local_qvm has been deprecated.\n"
+                                     "Please use pyquil.api.local_forest_runtime instead."))
+    with local_forest_runtime() as (qvm, quilc):
+        yield (qvm, quilc)
+
+
+def _port_used(port: int):
+    """Check if a (TCP) port is listening on localhost.
+    Returns ``True`` if a process is listening on the specified port, ``False`` otherwise.
+
+    :param port: TCP port which should get checked.
+    """
+    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    try:
+        s.connect(('127.0.0.1', port))
```

Comment:
We ought to make the host configurable, since it is configurable in `local_forest_runtime`.

Reply:
Just to clarify, I may got that wrong.

The `host` parameter in `local_forest_runtime` defines which address to listen on,
so if we try to connect to `127.0.0.1` we should always hit what we want to hit no?

More than happy though to make that configurable and pass `127.0.0.1` from the `local_forest_runtime` as a parameter, I think that'd make sense!

Reply:
I added the host parameter for the `_check_port` function, and `local_forest_runtime` passes now `127.0.0.1` to them. If this is not what's needed happy to revert it back/change it.